expertise/create_dataset.py

```diff
@@ -48,7 +51,10 @@ def get_paper_notes(self, author_id, dataset_params):
             note_ids = [e.head for e in bids if e.label in ['Very High', 'High']]
             return [n for n in self.openreview_client.get_notes_by_ids(ids=note_ids) if n.invitation == paper_invitation]
 
-        return openreview.tools.iterget_notes(self.openreview_client, content={'authorids': author_id})
+        if self.get_api_version() == 1:
+            return openreview.tools.iterget_notes(self.openreview_client, content={'authorids': author_id})
+        elif self.get_api_version() == 2:
+            return openreview.tools.iterget_notes(self.openreview_client, content={'authorids': { 'value': [author_id] }})
```

Comment:
no need to make this change, the API 2 supports `content={'authorids': author_id}`. 

Check this call from the dev site:

`notes = list(openreview.tools.iterget_notes(dev_client, content={ 'authorids': '~Melisa_Bok1'}))`